openupgrade_scripts/scripts/hr_recruitment/16.0.1.1/noupdate_changes.xml

```diff
@@ -7,7 +7,9 @@
     <field name="description">The user interacting with the application as interviewer don't need any specific access. They'll have access thanks to their interviewer assignation.</field>
   </record>
   <record id="email_template_data_applicant_congratulations" model="mail.template">
-    <field name="body_html" type="html">
+  <!--    The default template from version 15.0 should still work in version 16.0
+          No need to erase user data -->
+    <!-- <field name="body_html" type="html">
```

Comment:
If the template has changed, although the old one works, we should overwrite it, so we benefit from the new format/redaction/etc introduced in the new version. I know that this is counter-productive for customized templates, but it's not the usual (most of the installations keep the standard templates). One fine-tuning would be to detect if we have the original, and only overwrite in such case, but that's something very hard to be implemented, so the decision was some time ago to always overwrite.

Reply:
Thanks for clarifying :ok_hand:

Reply:
@legalsylvain is this rationale included somehow in the migration scripts guide you wrote? If not, it would be interested to be added.